### PR TITLE
Polish (pl) locale

### PR DESCRIPTION
**Pull request by splewako**
Please add Polish (pl) locale.

### CODE REVIEW

**Comment by whimboo** [comment]
@splewako thank you for the Polish translation! Given that I do not speak this language we would need someone to proof it. @stasm do you know who could help here? Thanks.

**Comment by stasm** [comment]
Hey @whimboo -- @splewako is the owner of the Polish localization of Firefox :)  I don't think we need to involve more people.  If @splewako feels otherwise I'm sure he can recommend a good reviewer for this work.

**Comment by whimboo** [comment]
Oh, I see. Thanks for this information. I wasn't aware of that. So I will leave it up for him to decide.

@splewako btw. I landed a change recently for the crashme extension which caused some changes to DTD entities. So the patch needs to be updated.

**Comment by splewako** [comment]
Others that come to my mind: @piotrdrag, @adrianer, @zbraniecki, @Witia1 but I wouldn't block on waiting  for review. If something will come up, feedback or updates to en-US I will update my fork and reopen this pull request.

**Comment by whimboo** [comment]
@splewako, is the comment from @piotrdrag important to have? If yes, can you please update it? I want to release a new version of NTT soon and it would be good to see this locale added. Thanks.

File: extension/chrome/locale/pl/nightly.properties [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+extensions.{8620c15f-30dc-4dba-a131-7c5d20cf4a29}.description=Użyteczne narzędzia dla testerów wersji Nightly.
+
+nightly.notextbox.message=Musisz zaznaczyć pole tekstowe zanim użyjesz tej funkcji.
```
Comment (splewako) [inline extension/chrome/locale/pl/nightly.properties]: Fixed

**Comment by splewako** [comment]
Yes, sorry, I was bit busy last days.
@whimboo, it should be ready to merge now.

**Comment by whimboo** [comment]
Thanks @splewako for the update. I will get it merged in a moment.

**Pull request merged**